data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApi.java

```diff
@@ -146,9 +146,9 @@ private void addBeaconHandlers(
     app.get(
         BeaconChainHeadHandler.ROUTE, new BeaconChainHeadHandler(chainStorageClient, jsonProvider));
     app.get(BeaconStateHandler.ROUTE, new BeaconStateHandler(chainStorageClient, jsonProvider));
-    // TODO: not in Minimal or optional specified set - some are similar to lighthouse
-    // implementation
-    handlers.add(new BeaconBlockHandler(chainStorageClient, historicalChainData));
+    app.get(
```

Comment:
nit: we should probably alpha order the endpoints by class name now, so they're easier to find later..